runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/discovery/CRCBeanType.java

```diff
@@ -23,7 +23,6 @@ public String getServerAdapterTypeId(String version) {
 	protected String getFullVersion(MinishiftVersions props) {
 		if (props != null
 				&& isSupported(props)
-				&& props.getMinishiftVersion() != null
```

Comment:
When doing crc version, the minishift key is not in the result, because this is crc and not minishift ;)